plugin/README.md

```diff
@@ -0,0 +1 @@
+../README.md
```

Comment:
Hm, I did a quick Google AI search and [looks like symlinks won't work like we expect in NPM](https://www.google.com/search?q=do+symlinks+work+for+readme+for+npm&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIGCAEQRRhA0gEINDMzNGowajeoAgCwAgA&sourceid=chrome&ie=UTF-8&udm=50&fbs=AIIjpHxU7SXXniUZfeShr2fp4giZ59Aj-dkSgmXWKpa2HWaBZINN7uSbO5j50upEZ-rgmOPkCUSfxswaNwgexZczMvZVCIr5KE3szbxketd0nhj1wPQJ8dYzOpC6dW1JXxA4o_Vhv6C9MVt9c7EgSEUfUlWHzHwMklA60btTWmheFBtbd0QhvrztO4xdNyQhPIKJg7euIc4ilV4C_JgPiuaUsLt_RrP87A&ved=2ahUKEwjOkIKQ9aGRAxV7MVkFHcKkIX8Q0NsOegQIJRAA&aep=10&ntc=1). 

Instead, I'd say the workaround to this is moving the `README` from the root repo to the plugin, then update the root README to be a simlink the the plugin readme instead